validator/project_validator.go

```diff
@@ -1213,6 +1213,12 @@ func validateTaskDependencies(project *model.Project) ValidationErrors {
 					Message: fmt.Sprintf("Task '%s' depends on non-patchable task '%s'. Neither will run in patches", task.Name, dep.Name),
 				})
 			}
+			if utility.FromBoolPtr(dependent.Disable) && !utility.FromBoolPtr(task.Disable) {
+				errs = append(errs, ValidationError{
+					Level:   Warning,
+					Message: fmt.Sprintf("Task '%s' depends on disabled task '%s'. Neither will run", task.Name, dep.Name),
```

Comment:
"Neither will run" isn't actually the behavior we want.

Reply:
hmm im not sure what should happen in that case then

Reply:
We can zoom if you want about this. If taskA depends on taskB, I think it should treat taskB as though it doesn't exist entirely; users want to use "disable" as a way to essentially "comment out" a task without having to make tons of small updates wherever it's referenced (like dependencies).

I think this can be handled here: https://github.com/evergreen-ci/evergreen/blob/b41b8975e9f4bd0afc0251c0fa83e1bb377e4f95/model/dependencies.go#L179 we should check if it's disabled before adding it.